ams_app/templates/ams_app/base.html

```diff
@@ -0,0 +1,44 @@
+{% load staticfiles %}
+<!DOCTYPE html>
+<html>
+<head>
+    <title>
+      {% block title %}{% endblock %}
+    </title>
+      <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
It can happen at either place. I just found out it's preferable to put JS files at the end of the body though. https://stackoverflow.com/questions/2451417/whats-pros-and-cons-putting-javascript-in-head-and-putting-just-before-the-bod

Reply:
Yup, wanted you to look it up 😉